### DISCUSSION TITLE
Why doesn't a layout inherit config.kdl?

### DISCUSSION THREAD

**Discussion by andradei** [zellij-org/zellij#2363] [Q&A]
I created a `~/.config/zellij/config.kdl` and also a `~/.config/zellij/layouts/my_layout.kdl`

When I run `zellij --layout ~/.config/zellij/layouts/my_layout.kdl` it doesn't load the `theme="pencil-light"` I put in `config.kdl`.

Is that the correct behavior? Are layouts a complete override of `config.kdl` (meaning I'll have to repeat theme, keybindings, etc., in every new layout)?

**Comment by andradei** [comment]
Apparently this is an issue of user experience, a bug, or it's not in the docs (might be there and I haven't found it)
https://github.com/zellij-org/zellij/issues/1894

**Comment by scottames** [comment]
Same thing here. I would love to be able to dry up my config. Having to repeat portions of or the entire `~/.config/zellij/config.kdl` into `~/.config/zellij/layouts/my_layout.kdl` seems like an anti-pattern. Having a possible 'include' or 'import' in the layout kdl would be another potential solve.
